R/pkg/R/functions.R

```diff
@@ -3720,3 +3720,22 @@ setMethod("current_timestamp",
             jc <- callJStatic("org.apache.spark.sql.functions", "current_timestamp")
             column(jc)
           })
+
+#' @details
+#' \code{from_csv}: Parses a column containing a CSV string into a Column of \code{structType}
+#' with the specified \code{schema}.
+#' If the string is unparseable, the Column will contain the value NA.
+#'
+#' @rdname column_collection_functions
+#' @param schema a DDL-formatted string
+#' @aliases from_csv from_csv,Column,character-method
+#'
+#' @note from_csv since 3.0.0
+setMethod("from_csv", signature(x = "Column", schema = "character"),
+          function(x, schema, ...) {
```

Comment:
can you add to the doc for `...` (in column_collection_functions) to indicate the use options for this function? if there is anything new?

Reply:
@felixcheung I added a doc but when I run `run-tests.sh`, it outputs the warning:
```
* checking Rd \usage sections ... WARNING
Duplicated \argument entries in documentation object 'column_collection_functions':
  ‘schema’

Functions with \usage entries need to have the appropriate \alias
entries, and all their arguments documented.
The \usage entries must correspond to syntactically valid R code.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```
Is it ok? If it isn't, any ideas what can cause it.

Reply:
no no, this will break - I am referring to find the original doc `@rdname column_collection_functions` that has `...` already documented, and then add this in

Reply:
here https://github.com/apache/spark/blob/d2bfd9430f05d006accdecb6a62ed659fbd6a2f8/R/pkg/R/functions.R#L199